modules/graceful-shutdown.adoc

```diff
@@ -59,15 +59,13 @@ Shutdown scheduled for Mon 2021-09-13 09:36:29 UTC, use 'shutdown -c' to cancel.
 ----
 +
 Shutting down the nodes using one of these methods allows pods to terminate gracefully, which reduces the chance for data corruption.
+
+. Optional: Adjust the shut down time to be longer for large-scale clusters:
```

Comment:
If "shut down" was used as a verb here, this would be fine. It's supposed to be "shutdown" for adjectives.